b2dEditor.py

```diff
@@ -0,0 +1,347 @@
+
+return_json = ""
+
+background = None
+
+def set_background():
+    global background
+
+    image_file_name = filedialog.askopenfilename(filetypes=[("PNG files", "*.png")], initialdir="~")
+    if image_file_name == "" or image_file_name == ():
+        return
+    background = pygame.image.load(image_file_name)
+    background.convert_alpha()
+    background.fill((255, 255, 255, 125), None, pygame.BLEND_RGBA_MULT)
+    background = pygame.transform.scale(background, (edit_area.rect[2] - 50, edit_area.rect[3] - 50))
+
+if "--help" in sys.argv:
+    print("\nUsage: python3 b2dEditor.py [--help] [--resolution <selection>]\n\n"
+          "--help:       Prints this help text\n"
+          "--resolution: Sets application resolution to selected option: \n"
+          "     1: 800x600\n"
+          "     2: 1200x900\n"
+          "     3: 1800x1350\n")
+    sys.exit(0)
+
+if "--resolution" in sys.argv:
+    resolution_arg = sys.argv[sys.argv.index("--resolution") + 1]
+    if resolution_arg == "1": resolution = (800, 600)
+    elif resolution_arg == "2": resolution = (1200, 900)
+    elif resolution_arg == "3": resolution = (1800, 1350)
+
+shapes = []
+current_shape = 0
+
+shapes.append([])
+
+buttons = []
+
+quit = False
+selected = None
+Tk().withdraw()
+
+def dump_node_json():
+    global return_json, quit
+    if __name__ == "__main__":
+        export_file_name = filedialog.asksaveasfilename(defaultextension=".json", filetypes=[("JSON files", "*.json")], initialdir="~")
+        if export_file_name == "" or export_file_name == ():
+            return
+
+    json_file = {}
+
+    rigid_body = {}
+    polygons = []
+
+    for i in range(len(shapes)):
+        pos_root_list = []
+        for j in shapes[i]:
+
+            pos_dict = {}
+
+            pos_dict["x"] = j.pos[0] / edit_area.rect[2]
+            pos_dict["y"] = j.pos[1] / edit_area.rect[3]
+
+            pos_root_list.append(pos_dict)
+
+        polygons.append([pos_root_list])
+
+    origin_pos = {}
+    origin_pos["x"] = origin.pos[0] / edit_area.rect[2]
+    origin_pos["y"] = origin.pos[1] / edit_area.rect[3]
+    rigid_body["origin"] = origin_pos
+    rigid_body["polygons"] = polygons
+    rigid_body["circles"] = []
+    rigid_body["shapes"] = []
+    json_file["rigidBody"] = rigid_body
+
+    if __name__ == "__main__":
+        with open(export_file_name, "w") as export_file:
+            json.dump(json_file, export_file, indent=2, sort_keys=True)
+    else:
+        return_json = json_file
+        quit = True
+
+def load_node_json():
+    import_file_name = filedialog.askopenfilename(filetypes=[("JSON files", "*.json")], initialdir="~")
+    if import_file_name == "" or import_file_name == ():
+        return
+
+    with open(import_file_name, "r") as import_file:
+        json_file = json.load(import_file)
+
+
+    for i in json_file["rigidBody"]["polygons"]:
+        for j in i:
+            for k in j:
+                x = int(k["x"] * edit_area.rect[2])
```

Comment:
Please more descriptive iterator names, especially for `k`. Even if you don't actually use `i` or `j` please give them more informative names